CHANGELOG.md

```diff
@@ -2,8 +2,43 @@
+
+  - `createPaymentMethod`'s `billingDetails` and `shippingDetails` parameters no longer accept the `addressPostalCode`, `addressCity`, `addressCountry`, `addressLine1`, `addressLine2`, or `addressState` keys. Instead, they accept an `address` object containing: `city`, `country`, `line1`, `line2`, `postalCode`, `state`.
+  - `confirmPayment`'s `billingDetails` and `shippingDetails` parameters no longer accept the `addressPostalCode`, `addressCity`, `addressCountry`, `addressLine1`, `addressLine2`, or `addressState` keys. Instead, they accept an `address` object containing: `city`, `country`, `line1`, `line2`, `postalCode`, `state`.
+  - `Card.Token` is now `Token.Result`
+  - `Card.Brand` is now `Token.CardBrand`
+  - `Card.TokenType` is now `Token.Type`
+  - `Card.BankAccount` is now `Token.BankAccount`
+  - `Card.Params` is now `Token.Params`
+  - `CardFormView.Names` is now `CardFormView.FieldName`
+  - `CardFieldInput.Names` is now `CardFieldInput.FieldName`
+  - `ApplePayButtonComponent.Styles` is now `ApplePayButtonComponent.Style`
+  - `ApplePayButtonComponent.Types` is now `ApplePayButtonComponent.Type`
+  - `PaymentMethod` is now `PaymentMethod.Result`
+  - `PaymentIntent` is now `PaymentIntent.Result`
+  - `SetupIntent` is now `SetupIntent.Result`
+  - `Card.Token` is now `Token`
+  - `CreateTokenParams` is now `Token.CreateParams`
+  - `BankAcccountHolderType` is now `Token.BankAcccountHolderType`
+  - `ThreeDSecureConfigurationParams` is now `ThreeDSecure.ConfigurationParams`
+  - `PaymentMethodCreateParams.Params` is now `PaymentMethod.CreateParams`
+  - `PaymentMethodCreateParams.Options` is now `PaymentMethod.ConfirmOptions`
+  - `CreateTokenParams` is now `Token.CreateParams`
+  - `ConfirmSetupIntent.Params` is now `SetupIntent.ConfirmParams`
+  - `ConfirmSetupIntent.Options` is now `SetupIntent.ConfirmOptions`
+  - The `confirmPayment` now accepts the `PaymentMethod.ConfirmParams` (same type, just renamed).
+  - `BillingDetails` type is now exported directly, instead of under the `PaymentMethodCreateParams` object.
+  - `BillingDetails` no longer includes `addressPostalCode`, `addressCity`, `addressCountry`, `addressLine1`, `addressLine2`, or `addressState` keys. Instead, it includes an `address` object containing: `city`, `country`, `line1`, `line2`, `postalCode`, `state` keys.
+  - `ShippingDetails` no longer includes `addressPostalCode`, `addressCity`, `addressCountry`, `addressLine1`, `addressLine2`, or `addressState` keys. Instead, it includes an `address` object containing: `city`, `country`, `line1`, `line2`, `postalCode`, `state` keys.
+  - `presentGooglePy` now accepts `GooglePay.PresentParams`
+  - `GooglePay.PresentGooglePayParams` is now `GooglePay.PresentParams`
+  - `GooglePay.PresentGooglePayType` is now `GooglePay.PresentType`
+  - `GooglePay.IsGooglePaySupportedParams` is now `GooglePay.IsSupportedParams`
+  - Removed `GooglePay.SetupIntentParams`
```

Comment:
Can you not do a SetupIntent with GooglePay anymore?

Reply:
you can, the type is just removed since it was unnecessary. It's been replace with `PresentParams`. See ``presentGooglePay` now accepts `GooglePay.PresentParams``